Yatima/Datatypes/Univ.lean

```diff
@@ -36,7 +36,15 @@ inductive Univ where
   | var   : Name → Nat → Univ
   deriving BEq, Inhabited
 
```

Comment:
Do we still want to namespace to `Yatima.Univ`?

Reply:
Yeah it's already done in this PR